### DISCUSSION TITLE
Towel 2.0 proposal

### DISCUSSION THREAD

**Discussion by egregors** [vldc-hq/vldc-bot#317] [Ideas]
❓ _Viele Wenig machen ein Viel._

Suggest solutions!

**Comment by ticketsvl** [comment]
As far as, ChatGPT has been already used in this project, my suggestion to continue so. Just need to add correct prompt for it with result scoring (1 to 10 or 1 to 100). Put examples there with spam messages and correct answers. Bad thing, that each new user/bot will cost some $.

**Reply by C-Pro** [reply]
Yep let robot decide who is worthy to be part of VLDC! Much simpler self-hosted model will do. But can use 3rd party API as a temporary solution.

**Reply by C-Pro** [reply]
Smth like this https://github.com/vldc-hq/vldc-bot/pull/318

**Comment by ticketsvl** [comment]
It seems like this is not good enough. So, look at bots behavior. The next message after greeting is an adv. Means, that nyan should monitor this message as well.

**Reply by C-Pro** [reply]
We can ask some question and ask to include it in introduction, but I think it is too complicated for most meatbags too.
I am reluctant to remove the bio requirement and resorting to quiz only. Это же наши скрепы!
